agreement_project/models/agreement.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class Agreement(models.Model):
+    _inherit = "agreement"
+
+    task_count = fields.Integer('# Tasks',
+                                compute='_compute_task_count')
+
+    @api.multi
+    def _compute_task_count(self):
+        for ag in self:
+            count = self.env['project.task'].search_count(
+                [('agreement_id', 'in', self.ids)])
```

Comment:
` [('agreement_id', 'in', ag.ids)]` or `[('agreement_id', '=', ag.id)]`

@ygol  @max3903 `self.ids` is correct it's call one time in form view, but if we used `for loop` then use loop variable.

Reply:
@nikul-serpentcs Can you make a pull request please?